sdks/java/core/src/test/java/org/apache/beam/sdk/values/TypedPValueTest.java

```diff
@@ -77,14 +78,10 @@ public void testUntypedOutputTupleTagGivesActionableMessage() {
 
     thrown.expect(IllegalStateException.class);
     thrown.expectMessage("No Coder has been manually specified");
-    thrown.expectMessage(
-        containsString("Building a Coder using a registered CoderFactory failed"));
-    thrown.expectMessage(
-        containsString("Building a Coder from the @DefaultCoder annotation failed"));
-    thrown.expectMessage(
-        containsString("Building a Coder from the fallback CoderProvider failed"));
+    thrown.expectMessage("Building a Coder using a registered CoderFactory failed");
```

Comment:
This error message is not even close to actionable. We have to find a way to tell the user what they can do to improve things.